pkg/resource/alias/hooks.go

```diff
@@ -191,14 +214,16 @@ func (rm *resourceManager) setResourceAdditionalFields(
 	exit := rlog.Trace("rm.setResourceAdditionalFields")
 	defer exit(err)
 
-	eic_err := rm.getFunctionEventInvokeConfig(ctx, ko)
-	if eic_err != nil {
-		return eic_err
+	// To set Asynchronous Invocations for the function's alias
+	err = rm.getFunctionEventInvokeConfig(ctx, ko)
+	if err != nil {
+		return err
```

Comment:
I guess this change doesn't belong into this PR, because it's related to event invoke config. Either we put everything together in one PR, or we actually split in two (both the changes to this file are added in both PRs)

Reply:
This change is just related to dividing the big `setResourceAdditionalFields` function into smaller parts. I had to do the changes for `ProvisionedConcurrency`, so I branched out `EventInvokeConfig` code too into separate function. I can keep the code for `FunctionEventInvokeConfig` as it is and make changes for `ProvisionedConcurrency` only in this PR.